db/tables/operations/select.py

```diff
@@ -35,6 +35,31 @@ def reflect_table_from_oid(oid, engine, connection_to_use=None):
     return reflect_table(table_name, schema, engine, connection_to_use=connection_to_use)
 
 
+def reflect_tables_from_oids(oids, engine, connection_to_use=None):
```

Comment:
Why not combine this function with the previous one? I think you should either loop through the singular version in this plural version, or (my preference) modify the singular one to call this one with appropriate arguments.